doc/GETSTARTED.md

```diff
@@ -1,88 +1,102 @@
-  mkdir build  && cd build  #create and navigate to the build directory
-  cmake ..  # watch output to see if everything needed is included
-  #  optionally run "ccmake .." to get an overview of the available build settings (needs cmake-curses-gui)
-  cmake --build build -- -j5
-  ```
+```sh
+cd libelektra  #navigate to libelektra
+mkdir build  && cd build  #create and navigate to the build directory
+cmake ..  # watch output to see if everything needed is included
+#  optionally run "ccmake .." to get an overview of the available build settings (needs cmake-curses-gui)
+cmake --build build -- -j5
+```
 
-  Optionally you can also run tests, see [here for more information](/doc/TESTING.md):
+Optionally you can also run tests, see [here for more information](/doc/TESTING.md):
 
-  ```sh
-  cmake --build build --target run_nokdbtests
-  ```
+```sh
+cmake --build build --target run_nokdbtests
+```
 
-  With these commands you will be able to run the "Hello World!" example, but usually you will need to use some of the [plugins](/src/plugins/README.md), tools and bindings of Elektra. Please take a look at the more detailed [compiling documentation](/doc/COMPILE.md). After you completed building Elektra on your own, you can execute these commands to install Elektra (please check the [installation documentation](/doc/INSTALL.md) for the many available packages):
+With these commands you will be able to run the "Hello World!" example, but usually you will need to use some of the [plugins](/src/plugins/README.md), tools and bindings of Elektra. Please take a look at the more detailed [compiling documentation](/doc/COMPILE.md). After you completed building Elektra on your own, you can execute these commands to install Elektra (please check the [installation documentation](/doc/INSTALL.md) for the many available packages):
 
-  ```sh
-  sudo make install
-  sudo ldconfig #optional: check installation documentation for more information
-  ```
+```sh
+sudo make install
+sudo ldconfig #optional: check installation documentation for more information
+```
 
-  [Installation documentation](/doc/INSTALL.md) contains further information about available packages.
+[Installation documentation](/doc/INSTALL.md) contains further information about available packages.
 
-  Optionally you can also run tests to verify the installed Elektra, see [here for more information](/doc/TESTING.md):
+Optionally you can also run tests to verify the installed Elektra, see [here for more information](/doc/TESTING.md):
 
-  ```sh
-  kdb run_nokdbtests
-  ```
+```sh
+kdb run_nokdbtests
+```
 
-- Hello World!
+## Docker
 
-  Start with your very first Elektra application in C and follow these steps:
-  [Hello World!](/doc/tutorials/hello-elektra.md)
+In order to ease the above process, we also provide [up-to-date docker images](https://hub.docker.com/u/elektra) which you can use to try docker without having to install and run docker on your local machine.
```

Comment:
This is very misleading. If you want to get started with development on Elektra you most likely need to do what is written above. I suggest you following change: write in the beginning of the GETSTARTED.md document a clarification for whom the document is intended and who should read which other document.

Reply:
True, maybe we could instead create a Docker image which sets up all requirements and offer a ready to develop environment?

Reply:
Maybe there is another barrier. For me it was not clear that this readme is only meant for developers. Maybe we could make a seperate developing readme in /dev?

Reply:
> For me it was not clear that this readme is only meant for developers.

This should be an easy fix: Please write the scope of the GETSTARTED.md in the beginning.

> I think for an enduser they should only read the install readme or spin up a container, which is much quicker.

For endusers we also have doc/INSTALL.md

I agree Docker could be mentioned there.